.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,52 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: 'The name of the tarantool build artifact'
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: 'Clone the expirationd module'
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/expirationd
+
+      - name: 'Download the tarantool build artifact'
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+      # TODO(ylobankov): Just in case remove tarantool that could be previously
+      # installed on the runner. Relevant to self-hosted runners. Remove this
+      # step when only GitHub-hosted runners are used for integration testing.
+      - run: sudo apt-get -y purge 'tarantool*'
+      - name: 'Install tarantool'
+        # TODO(ylobankov): Install package dependencies. Now we're lucky: all
+        # dependencies are already there.
+        run: sudo dpkg -i tarantool*.deb
+
+      - name: 'Install expirationd'
+        run: tarantoolctl rocks make
+
+      - name: 'Install test requirements'
+        run: tarantoolctl rocks install luatest 0.5.6
```

Comment:
Please make a separate target in Makefile for requirements installation (see https://github.com/tarantool/expirationd/blob/master/.github/workflows/fast_testing.yaml#L49-L56) and let's use `make deps` in workflows.

Reply:
Well, I need the only test requirement here - luatest. I don't think that we need to install luacheck, luacov, luacov-coveralls for such kind of testing that is done here. IMHO, It is redundant. So I decided to install exactly what I need. But for the fast_testing.yml workflow, creating a separate target for requirements installation will make sense.